packages/next-server/package.json

```diff
@@ -42,8 +42,8 @@
   "devDependencies": {
     "@taskr/clear": "1.1.0",
     "@taskr/watch": "1.1.0",
-    "@types/react": "16.7.13",
-    "@types/react-dom": "16.0.11",
+    "@types/react": "16.8.4",
+    "@types/react-dom": "16.8.2",
```

Comment:
Shouldn’t React and React-Dom be the same version? I’ve had bugs with that before...

Reply:
Those are the the newest versions available for both of them. `@types/react` had a bug until I upgraded it to `16.8.4`